sormas-backend/src/main/java/de/symeda/sormas/backend/survey/SurveyFacadeEjb.java

```diff
@@ -194,15 +210,24 @@ private SurveyDto toDto(Survey source) {
 		return target;
 	}
 
+	public static SurveyReferenceDto toReferenceDto(Survey entity) {
+
+		if (entity == null) {
+			return null;
+		}
+
+		return new SurveyReferenceDto(entity.getUuid(), entity.getDisease(), entity.getName());
+	}
+
 	public SurveyReferenceDto convertToReferenceDto(Survey survey) {
 		return toSurveyReferenceDto(survey);
 	}
 
 	public static SurveyReferenceDto toSurveyReferenceDto(Survey survey) {
-		return new SurveyReferenceDto(survey.getUuid(), survey.getName());
+		return new SurveyReferenceDto(survey.getUuid(), survey.getDisease(), survey.getName());
```

Comment:
please remove the disease

The caption will be displayed on select lists filtered by disease, so it is redundant

Reply:
removed